x-team/xp-ui
# Add summary and learn more to JobCard

**Release Type:** *Non-Breaking Feature* 

Fixes https://x-team-internal.atlassian.net/browse/XP-3360

## Description

Change the `JobCard` component to display the Job's summary field followed by `Learn more >>` link that fires back a `onLearnMore` callback.

## Checklist


## Related PRs

branch | PR
------ | ------
`XP-3360-job-card-summary` | [XP](https://github.com/x-team/xp-registraion/pull/79)

## Steps to Test or Reproduce

1. Run storybook
2. Go to `JobCard`
3. Observe that card shows summary, followed by `... Learn more >>`
4. Observe that clicking the Learn more prints callback event to console

## Impacted Areas in Application

`components/ui/JobCard` component

## Screenshots

![image](https://user-images.githubusercontent.com/15269623/66624165-db1fc600-eba3-11e9-9d04-e2977e3e0a48.png)
